policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/child_dependent_deduction.yaml

```
@@ -0,0 +1,11 @@
+description: Virginia provides a itemized deduction for child and dependent care expense. $3,000 for one dependent, and $6,000 for two or more dependents.
```

Descriptions omit values, and we can generally assume deductions are itemized unless they're the standard deduction
```suggestion
description: Virginia provides a deduction for child and dependent care expenses, up to this amount, or double for more than one dependent.
```